packages/garbo/src/lib.ts

```diff
@@ -1150,3 +1151,8 @@ export function improvedStats(thing: Item | Effect): Stat[] {
 export function improvesAStat(thing: Item | Effect): boolean {
   return improvedStats(thing).length > 0;
 }
+
+export const doCyberRealmZone3 =
+  get("_cyberFreeFights") === 0 &&
+  have($item`zero-trust tanktop`) &&
+  garboValue(CyberRealm.zone3Rewards()) > get("valueOfAdventure") * 9;
```

Comment:
we don't want ground-level variables that call garboValue

Reply:
(in other words, find a way to make this whole thing a function. I would combine _cyberFreeFights with the zone 3 fights pref)